### ISSUE TITLE
Possible error in `dcm_write`

### ISSUE THREAD

**Issue by Dale-Black** [JuliaHealth/DICOM.jl#85]
I wrote this out on [Zulip](https://julialang.zulipchat.com/#narrow/stream/274208-helpdesk-.28published.29/topic/Help.20with.20DICOM.2Ejl.20.60dcm_write.60) but I realized this might actually be a bug in `dcm_write` the more I am looking into it 

Any idea why a tag `(0x005e, 0x0040)` is being created after creating a DICOM file using `dcm_write`?

**Comment by notZaki** [comment]
It could be a bug. 

What were the types for `array` and `file`? The `dcm_write` function doesn't currently do any sanity checks, so a possible reason could be that the replaced values don't have the same type as the old PixelData and InstanceNumber values.

**Comment by Dale-Black** [comment]
It looks like it does have something to do with the `dcm[tag"Pixel Data"] = array` operation. When I comment out that line, the code runs fine and I can indeed `write` that to a new DICOM file and then load it using `dcm_parse`. I don't think it's the array type that is causing problems though since they have the same types. Could it be a problem with replacing the original `Pixel Data` with a different size array for the new `Pixel Data`? 

![image](https://user-images.githubusercontent.com/62632960/156245706-1abf2345-97d7-4c8c-bdee-9603d960e5db.png)

**Comment by notZaki** [comment]
Yup, the `Rows` and `Columns` fields will have to be updated if the size is different. I can never remember which of these should be 440 and which should be 640. Maybe see which combination gives a proper looking image.

**Comment by Dale-Black** [comment]
That solved it
```julia
dcm[tag"Rows"] = size(array, 1)
dcm[tag"Columns"] = size(array, 2)
```

**Comment by Dale-Black** [comment]
I am guessing there is a way to check for something like this when using `dcm_write` so if I ever get enough time I will try to create a PR to do just that
